.github/CODEOWNERS

```diff
@@ -6,5 +6,9 @@
 /tasks/               @SalesforceFoundation/release-engineering-reviewers
 
 # Packaged Metadata
-/force-app/main/default/labels  @microveldt @AndersonTarren
-/force-app/main/default/objects  @microveldt @AndersonTarren
+/force-app/main/default/labels   @AndersonTarren
```

Comment:
@AndersonTarren should we also have @lbeller and @smarovich on both of these?

Reply:
@jjbennett  - I think that's a smart idea.

Reply:
Done